app.rb

```diff
@@ -20,12 +22,38 @@ def initialize
     @compliance_data = ComplianceData.new
   end
 
+  set :assets, Sprockets::Environment.new(root)
+
+  configure do
+    assets.append_path File.join(root, 'assets', 'stylesheets')
+    assets.append_path File.join(root, 'assets', 'javascripts')
+
+    Sprockets::Helpers.configure do |config|
+      config.environment = assets
+      config.prefix      = '/assets'
+      config.digest      = true
+    end
+  end
+
+  helpers do
+    include Sprockets::Helpers
+  end
+
+  before do
+    cache_control :public, :must_revalidate, :max_age => 60
+  end
+
   get '/auth/myusa/callback' do
     halt(401, 'Not Authorized') unless env['omniauth.auth']
     session[:user_email] = env['omniauth.auth'].info.email
     redirect '/'
   end
 
+  get '/auth/logout' do
+    session[:user_email] = nil 
+    redirect "https://alpha.my.usa.gov/users/sign_out?continue=http://#{host_with_port}/"
```

Comment:
I'm pretty sure this is the wrong way to do it ('cause I did it). I'm looking for a more effective way to do this with less hardcoding. I'm not sure my.usa.gov is handling the continue parameter correctly.